portality/view/doaj.py

```diff
@@ -58,13 +58,21 @@ def news():
     news = blog.News.latest(app.config.get("NEWS_PAGE_NEWS_ITEMS", 20))
     return render_template('doaj/news.html', news=news, blog_url=app.config.get("BLOG_URL"))
 
+@blueprint.route("/widgets_demo")
+def widgets_demo():
+    return render_template('doaj/fqw_demo.html',
+                           env=app.config.get("DOAJENV"),
+                           widget_filename_suffix='' if app.config.get('DOAJENV') == 'production' else '_' + app.config.get('DOAJENV', ''),
+                           lcc_tree=lcc_jstree
+                           )
 
-# @blueprint.route("/widgets")
-# def widgets():
-#     return render_template('doaj/widgets.html',
-#                            env=app.config.get("DOAJENV"),
-#                            widget_filename_suffix='' if app.config.get('DOAJENV') == 'production' else '_' + app.config.get('DOAJENV', '')
-#                            )
+@blueprint.route("/widgets_demo_empty")
```

Comment:
This worked ok, but there were a couple of issues:

1. I got a JS error on load which didn't seem to block the page, but we should figure it out

![Screenshot from 2021-03-05 11-04-32](https://user-images.githubusercontent.com/856138/110107948-78842700-7da3-11eb-9094-46400f8d8dbd.png)


2. The page overflowed the window giving me a horizontal scroll.  This might be resolved when the styles of this are looked at.